homeassistant/components/unifi/device_tracker.py

```diff
@@ -146,61 +171,29 @@ def update_items(controller, async_add_entities, tracked):
     """Update tracked device state from the controller."""
     new_tracked = []
 
-    if controller.option_track_clients:
+    for client_id in controller.api.clients:
 
-        for client_id in controller.api.clients:
-
-            if client_id in tracked and tracked[client_id].entity_id:
-                LOGGER.debug(
-                    "Updating UniFi tracked client %s (%s)",
-                    tracked[client_id].entity_id,
-                    tracked[client_id].client.mac,
-                )
+        if client_id in tracked:
+            if tracked[client_id].registry_entry.disabled_by is None:
                 tracked[client_id].async_schedule_update_ha_state()
-                continue
+            continue
 
-            client = controller.api.clients[client_id]
+        client = controller.api.clients[client_id]
 
-            if (
-                not client.is_wired
-                and controller.option_ssid_filter
-                and client.essid not in controller.option_ssid_filter
-            ):
-                continue
-
-            if not controller.option_track_wired_clients and client.is_wired:
-                continue
-
-            tracked[client_id] = UniFiClientTracker(client, controller)
-            new_tracked.append(tracked[client_id])
-            LOGGER.debug(
-                "New UniFi client tracker %s (%s)",
-                client.name or client.hostname,
-                client.mac,
-            )
+        tracked[client_id] = UniFiClientTracker(client, controller)
+        new_tracked.append(tracked[client_id])
 
-    if controller.option_track_devices:
+    for device_id in controller.api.devices:
 
-        for device_id in controller.api.devices:
-
-            if device_id in tracked and tracked[device_id].entity_id:
-                LOGGER.debug(
-                    "Updating UniFi tracked device %s (%s)",
-                    tracked[device_id].entity_id,
-                    tracked[device_id].device.mac,
-                )
+        if device_id in tracked:
+            if tracked[device_id].registry_entry.disabled_by is None:
```

Comment:
I feel like a bunch of code might need this, let's add a new property to Entity:

```python
@property
def enabled(self):
    return self.registry_entry is None or self.registry_entry.disabled_by is None
```

Reply:
Will do!